styleguide/components/41-organisms/opening-hours-accordion/README.md

```diff
@@ -0,0 +1,25 @@
+# Opening hours accordion
+
+## When to use this component
+
+Use the opening hours accordion component to:
+
+* Show an overview of opening hours for a certain location that has one or multiple ways of being open, for instance, free entrance without appointment versus appointment only.
+* Include opening hours but clearly show the type of opening hours and show or hide them depending on if the opening hours are essential for all users or only necessary for a minority of users (progressive disclosure principle).
+* Offer opening hours for this week and for this month and what the opening hours are for each day but also have a quick way to show if the location is open today and what the opening hours are for today.
+
+The opening hours accordion is only to be used on detail pages of locations.
```

Comment:
just to be sure: that implies you can **never** use it on for example a contact page?

Reply:
That does seem counterproductive.
A lot of effort has been made to make the opening hours api and widget open source and cross platform so it can be used as much as possible and to avoid business adding plain text opening hours on contact pages.

Reply:
@hstandaert @delrueba I did not mean that it cannot be used on a contact page (for me, a contact page also has information about a location and opening hours mostly resemble there is some kind of physical location linked to a contact).

I will just update this and describe it as it can be used on (all) detail pages.